css/style.css

```diff
@@ -59,6 +59,8 @@ footer { margin: 1em; font-size: 0.75em; text-align: center; }
   clear: both;
 }
 .container > #sidebar {
+  height: 100vh;
```

Comment:
Thanks for the catch! Why would you need "height" here? Sorry if this is a silly question; I'm not familiar with css.

Reply:
Need to set the sidebar height to fill 100% of the viewport to make overflow take effect. Otherwise, the sidebar will only match the content height.